js/cards.js

```diff
@@ -91,7 +46,7 @@ export async function defineCardComponent() {
     //Returns an url of the form:
     //`https://res.cloudinary.com/${CLOUD_NAME}/image/upload/${RARITY}/${FILENAME}.png`
     getImageURL() {
-      return `${CLOUDINARY_URL}/q_auto/${this.data["Rarity Folder"]}/${this.data["Filename"]}.png`;
+      return `${CLOUDINARY_URL}q_auto/${this.data["Rarity Folder"]}/${this.data["Filename"]}.png`;
     }
 
     setupCardForGacha() {
```

Comment:
The gacha specific code might be nice to split out too to be consistent with the collection.js organization. We can do it later if we have time